### ISSUE TITLE
Editing code through the Github website is broken: commit button does not work and line numbers are missing

### ISSUE THREAD

**Issue by ProBackup-nl** [ghostery/ghostery-extension#809]
### Description

When editing "Code" on github, the extension blocks display of line numbering on the left of code.

### Expected Behavior

Line numbering displayed:
![Editing padavan-ng_readme eng txt at custom · zanezam_padavan-ng – (Private Browsing)](https://user-images.githubusercontent.com/515451/177786326-37cfaf7c-6db5-4967-a401-22208aadf1a0.png)

Console output:
- Security Error: Content at https://github.com/zanezam/padavan-ng/edit/custom/toolchain/readme.eng.txt may not load or link to moz-extension://9fe646ab-d1b3-2e46-95d2-74f9c985f372/js/content/inject.js.
- Content Security Policy: The page's settings blocked the loading of a resource at inline ("script-src"). content.js:38:10
- Source map error: Error: NetworkError when attempting to fetch resource. Resource URL: moz-extension://8a437308-be91-1345-b85c-c582f97f63bd/browser-polyfill.min.js Source Map URL: browser-polyfill.min.js.map

### Actual Behavior

No line numbering:
![Editing padavan-ng_readme eng txt at custom · zanezam_padavan-ng](https://user-images.githubusercontent.com/515451/177786468-bb7bd197-5dbe-4160-b9ae-ae01a60a8dd7.png)

Console output:
- Security Error: Content at https://github.com/zanezam/padavan-ng/edit/custom/toolchain/readme.eng.txt may not load or link to moz-extension://9fe646ab-d1b3-2e46-95d2-74f9c985f372/js/content/inject.js.

- Content Security Policy: The page's settings blocked the loading of a resource at inline ("script-src"). single-file-extension-frames.js:1:294

- Content Security Policy: The page's settings blocked the loading of a resource at inline ("script-src"). content.js:40:10

- Content Security Policy: The page's settings blocked the loading of a resource at inline ("script-src"). sandbox eval code:21:35

- Content Security Policy: The page's settings blocked the loading of a resource at data: ("media-src").

- Content Security Policy: The page's settings blocked the loading of a resource at inline ("script-src"). utils.js:33:9

- Content Security Policy: The page's settings blocked the loading of a resource at data:application/javascript;base64,dmFyI… ("script-src"). 2

- Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://github.githubassets.com/assets/vendors-node_modules_codemirror_lib_codemirror_js-1ea5dd45dd65.js. (Reason: CORS request did not succeed).

- None of the "sha512" hashes in the integrity attribute match the content of the subresource. readme.eng.txt

- Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://github.githubassets.com/assets/vendors-node_modules_github_mini-throttle_dist_index_js-node_modules_codemirror-contrib_lib_m-684a28-d63806260af3.js. (Reason: CORS request did not succeed).

- None of the "sha512" hashes in the integrity attribute match the content of the subresource. readme.eng.txt

- SingleFile is hooking the IntersectionObserver API to detect and load deferred images. single-file-hooks-frames.js:1:531

- Failed to get service worker registration(s): Storage access is restricted in this context due to user settings or private browsing mode. register-service-worker.ts:17:50

- Source map error: Error: NetworkError when attempting to fetch resource. Resource URL: moz-extension://8a437308-be91-1345-b85c-c582f97f63bd/browser-polyfill.min.js Source Map URL: browser-polyfill.min.js.map

### Steps to Reproduce

1. Goto https://github.com/zanezam/padavan-ng/edit/custom/toolchain/readme.eng.txt

Note: first login to github

### Versions

Ghostery version: 8.7.4
Database Version: 4ff20ff58c8d7d7df467a3a2464fa96e
Browser name: Firefox
Browser version: 78
Language: nl
OS: mac
Category: site_analytics
Allowed Trackers:
Blocked Trackers: Adobe Experience Cloud

**Comment by philipp-classen** [comment]
Thanks for reporting! The problem with the missing line numbers still exists. Also, committing changes through the Github UI does not work.

**Comment by philipp-classen** [comment]
Blocking `Adobe Experience Cloud` breaks the page. Since it is classified as "Site Analytics" (not "Essentials") it will also be blocked by default.

Until we have a fix, a workaround is to trust `Adobe Experience Cloud` only on Github:

![workaround-for-github](https://user-images.githubusercontent.com/1616509/182379539-e12aab7a-093a-43dd-901a-b6f455753ea4.png)

**Comment by philipp-classen** [comment]
It turns out to be a false-positive. URLs like the following are incorrectly classified as `Adobe Experience Cloud` site analytics:

https://github.githubassets.com/assets/vendors-node_modules_codemirror_lib_codemirror_js-9124701c2e90.js

In other words, we have to improve our regexp to fix the breakage.

**Comment by ProBackup-nl** [comment]
I can confirm that with 2022 August 10th update, version 8.7.6, line numbers are visible again and that no trackers are detected.
